$name__norm$-impl/src/main/scala/$package$/impl/$name__Camel$ServiceImpl.scala

```diff
@@ -24,4 +27,18 @@ class $name;format="Camel"$ServiceImpl(persistentEntityRegistry: PersistentEntit
     // Tell the entity to use the greeting message specified.
     ref.ask(UseGreetingMessage(request.message))
   }
+
+
+  override def greetingsTopic(): Topic[GreetingMessage] =
+    TopicProducer.singleStreamWithOffset {
+      fromOffset =>
+        persistentEntityRegistry.eventStream($name;format="Camel"$Event.Tag, fromOffset)
+          .map(ev => (convertEvent(ev), ev.offset))
+    }
+
+  private def convertEvent(helloEvent: EventStreamElement[$name;format="Camel"$Event]): GreetingMessage = {
+    helloEvent.event match {
+      case GreetingMessageChanged(msg) => GreetingMessage(helloEvent.entityId, msg)
```

Comment:
(see my comment above)
Outgoing message may have less or more info than internal message. Here, we fill the name with the `entityId` and the internal message doesn't need to carry it over.